File: Projeto1/app/src/main/java/com/example/a1528956/projeto1/MainActivity.java
package com.example.a1528956.projeto1;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.EditText;
import android.widget.TextView;

import br.com.projetocombustivel.controle.combustivelCotrole;

public class MainActivity extends AppCompatActivity {

    combustivelCotrole cn = new combustivelCotrole();
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void clique(android.view.View view){
        EditText gasolina = (EditText) findViewById(R.id.txtVlrGas);
        EditText etanol = (EditText) findViewById(R.id.txtVlrAlc);
        TextView textView = (TextView) findViewById(R.id.lblResultado);
        textView.setText(cn.Calcular(gasolina.getText().toString(), etanol.getText().toString()).getDescricao());
    }
}

File: AulaProjeto4/app/src/main/java/com/example/a1528956/aulaprojeto4/MainActivity.kt
package com.example.a1528956.aulaprojeto4

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnFGTS.setOnClickListener {
            // abrindo outra página
            val intent = Intent(this, FGTSActivity::class.java)
            startActivity(intent)
        }

        btnSeguroDesemprego.setOnClickListener{
            var intent = Intent(this, SegDesempregoActivity::class.java)
            startActivity(intent)
        }
        btnINSS.setOnClickListener{
            var intent = Intent(this, InssActivity::class.java)
            startActivity(intent)
        }
    }
}
